packages/manager/apps/key-management-service/src/modules/secret-manager/pages/dashboard/versions/Versions.page.tsx

```diff
@@ -54,10 +49,25 @@ const SecretVersionsPage = () => {
     },
   ];
 
+  const {
+    flattenData: versions,
+    error,
+    hasNextPage,
+    fetchNextPage,
+    sorting,
+    isLoading,
+    setSorting,
+    refetch,
+  } = useSecretVersions({ domainId, path: secretPathDecoded });
+
   if (error)
     return (
       <ErrorBanner
-        error={error}
+        error={{
+          data: {
+            message: error.message,
+          },
+        }}
```

Comment:
`useResourcesIcebergV2` screws up the error type, but you can use a typegard and get back on feets like I did here: https://github.com/ovh/manager/blob/0e0a7eb8113477a841502c2506a05338e26cdd20/packages/manager/apps/key-management-service/src/modules/secret-manager/pages/domains/Domains.page.tsx#L61